app/modules/elasticsearch/tasks.py

```diff
@@ -128,7 +128,55 @@ def load_encounters_index():
 """
 
 
+def load_sightings_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(SIGHTINGS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            sighting = Sighting(**result)
+            # Assign the elasticsearch document identify
+            sighting.meta.id = f'sighting_{sighting.id}'
+            # Save document to elasticsearch
+            sighting.save(using=current_app.elasticsearch)
+
+
+SIGHTINGS_INDEX_SQL = """\
+SELECT
+  oc."ID" AS id,
+  NULLIF((oc."DECIMALLATITUDE"::float || ',' || oc."DECIMALLONGITUDE")::text, ',') AS point,
+  start_time."DATETIME" AS start_time,
+  end_time."DATETIME" AS end_time,
+  -- First encounter genus
+  (
+    SELECT max("GENUS")
+    FROM "ENCOUNTER" en
+      RIGHT JOIN "OCCURRENCE_ENCOUNTERS" oe ON oe."ID_EID" = en."ID"
+    WHERE oe."ID_OID" = oc."ID") AS genus,
+  (
+    SELECT max("SPECIES")
+    FROM "ANNOTATION" an
+      RIGHT JOIN "ENCOUNTER_ANNOTATIONS" ea ON ea."ID_EID" = an."ID"
+      RIGHT JOIN "ENCOUNTER" en ON ea."ID_OID" = en."ID"
+      RIGHT JOIN "OCCURRENCE_ENCOUNTERS" oe ON oe."ID_EID" = en."ID"
+    WHERE oe."ID_OID" = oc."ID") AS species,
+  oc."GROUPBEHAVIOR" AS group_behavior,
```

Comment:
this whole set of properties can be removed/ignored:  `GROUPBEHAVIOR` thru `OBSERVER`, as well as `COMPLEXDATETIME`.  these have become cruft that are really only there for migration.   the one join-table that _is_ important is the one relating to CustomFields.... but this is a fairly complex join.  i am not sure this has been dealt with in elasticsearch yet.   might be worth talking about that more.

Reply:
I'm going to remove:
 - `GROUPBEHAVIOR`,
 - `GROUPCOMPOSITION`, 
 - `FIELDSTUDYSITE`, 
 - `INITIALCUE`, 
 - `SEASTATE`, 
 - `HUMANACTIVITYNEARBY`, 
 - `OBSERVER`

If we remove `COMPLEXDATETIME` there's no date time fields anymore for the sightings search, do we need to replace them with something else?

Reply:
> If we remove `COMPLEXDATETIME` there's no date time fields anymore for the sightings search, do we need to replace them with something else?

there is now ComplexDateTime class _in houston_, so the value needs to be added from there.  it is on both the Sighting and Encounter models as the `time` property.  (note, however, that it should be indexed in elasticsearch like the schemas return it from the api for example:

- `sighting.time` => `"2000-01-01T13:13:13+07:00"`   _and_
- `sighting.timeSpecificity` => `day`